src/OpenLoco/src/Drawing/FPSCounter.cpp

```diff
@@ -2,6 +2,7 @@
 #include "Drawing/SoftwareDrawingEngine.h"
 #include "Graphics/Colour.h"
 #include "Graphics/Gfx.h"
+#include "Localisation/Formatting.h"
 #include "Localisation/StringManager.h"
```

Comment:
For the files that you added a `Formatting.h` include to in this PR, did you verify the `StringManager.h` include is now obsolete? That seems to be the case here, for example.

Reply:
I removed the include here but I haven't really checked for trivial includes, it wouldn't make that big difference currently as Formatting.h includes StringManager.h anyway as it depends on the data types from there, this will be addressed with further reorganizations.